Docs 11944 am permissions (#188)
mulesoft/docs-monitoring: modules/ROOT/pages/am-permissions.adoc

> = Manage Permissions for Anypoint Monitoring

Reviewer: I am not sure whether there is more to this topic set, but from what I see in this PR, you could replace the existing Level-1 plus subtopic with a more specific Level-1 narrative:
```suggestion
= Manage Permissions for Anypoint Monitoring
```
See more below:

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> Anypoint Platform provides predefined permissions and roles that administrators can assign in the *Access Management* section of Anypoint Platform.
> Anypoint Monitoring has two product-specific permissions to control access for different types of users.
> 
> == Anypoint Monitoring Permissions
> 
> To control access to dashboards, reports, and metrics, use the following permissions for Anypoint Monitoring:

Reviewer: ```suggestion
The *Access Management* section of Anypoint Platform enables you as administrator to user control access to dashboards, reports, and metrics by providing two predefined product-specific permissions:
```

> * Anypoint Monitoring Administrator: +
> View, create, modify, and delete all content in Anypoint Monitoring.

Reviewer: ```suggestion
* Anypoint Monitoring Administrator. Allows users to view, create, modify, and delete all content in Anypoint Monitoring.
```

+

> * Anypoint Monitoring Viewer: +
> View content in Anypoint Monitoring. Users with this permission cannot modify any Anypoint Monitoring content.

Reviewer: ```suggestion
* Anypoint Monitoring Viewer. Allows users to view but not modify content in Anypoint Monitoring.
```
